cmd/root.go

```diff
@@ -10,9 +10,30 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const (
+	flagVerbose = "verbose"
+)
+
 var rootCmd = &cobra.Command{
 	Use:          "rotator",
 	SilenceUsage: true,
+	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
Might want to switch to the root cmd in go-misc -> https://github.com/chanzuckerberg/go-misc/blob/main/cmds/root.go#L17